mods/default/furnace.lua

```diff
@@ -292,6 +292,7 @@ minetest.register_node("default:furnace", {
 	on_construct = function(pos)
 		local meta = minetest.get_meta(pos)
 		meta:set_string("formspec", default.get_furnace_inactive_formspec())
+		meta:set_string("infotext", 'Furnace inactive\n(Item: Empty; Fuel: Empty)')
```

Comment:
This is also missing a translation.
You can't put the whole string into `S()` either, because that can't reuse the existing translations.

It'd be better to make the `furnace_node_timer` suggestion work.

Reply:
I just need to work out where to put it xD

Reply:
To allow for translation:
```
S("Furnace") .. S("inactive") .. "\n(" .. S("Item") .. ": " .. S("Empty") .. "; " .. S("Fuel") .. ": " .. S("Empty") .. ")"
```

Reply:
I've added `furnace_node_timer` and got it to work correctly.